pkg/controller/kogitojobsservice/kogitojobsservice_controller.go

```diff
@@ -141,13 +140,8 @@ const (
 	maxIntervalLimitRetryEnvKey       = "MAX_INTERVAL_LIMIT_RETRY"
 	backOffRetryDefaultValue          = 1000
 	maxIntervalLimitRetryDefaultValue = 60000
-	kafkaTopicNameJobsEvents          = "kogito-job-service-job-status-events"
 )
 
-var kafkaTopics = []services.KafkaTopicDefinition{
-	{TopicName: kafkaTopicNameJobsEvents, MessagingType: services.KafkaTopicOutgoing},
-}
-
```

Comment:
Same thing here related to my comment on Data Index

Reply:
I could double verify it but user only need to define `kafka.bootstrap.servers` property and these topics will be created automatically

Reply:
done